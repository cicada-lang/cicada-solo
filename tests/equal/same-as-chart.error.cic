same_as_chart! Nat [
  add1(add1(add1(0))),
  add1(add1(1)),
  add1(2),
  3,
  3,
  6,
  3,
  3,
]
