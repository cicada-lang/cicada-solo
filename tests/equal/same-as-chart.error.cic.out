I am expecting the following two values to be the same Nat.
But they are not.

from value at 4:
  3
to value at 5:
  6

previous:
- same_as_chart! Nat [ 3, 3, 3, 3, 3, 6, 3, 3 ]
